internal/server/http/handler_test.go

```diff
@@ -69,6 +71,59 @@ func TestImposterHandler(t *testing.T) {
 	}
 }
 
+func TestImposterHandler_Variables(t *testing.T) {
+	var headers = make(map[string]string)
+	headers["Content-Type"] = "application/json"
+
+	responseId1 := "test/testdata/imposters_variables/responses/gopher_1_response.json"
+	responseId2 := "test/testdata/imposters_variables/responses/gopher_2_response.json"
+	responseId1Variable1 := "test/testdata/imposters_variables/responses/gopher_1_1_response.json"
+	responseId1Variable2 := "test/testdata/imposters_variables/responses/gopher_1_2_response.json"
+
+	imposterFilePath := "test/testdata/imposters_variables/gopher_variables.imp.json"
+	imposterFile, _ := os.Open(imposterFilePath)
+	defer imposterFile.Close()
+	imposterBytes, _ := io.ReadAll(imposterFile)
+
+	var imposters []Imposter
+	err := json.Unmarshal(imposterBytes, &imposters)
+	assert.NoError(t, err)
+
+	var dataTest = []struct {
+		name             string
+		imposter         Imposter
+		url              string
+		expectedBodyPath string
+		statusCode       int
+	}{
+		{"valid imposter with id 1 in path", imposters[0], "/gophers/1", responseId1, http.StatusOK},
+		{"valid imposter with id 2 in path", imposters[0], "/gophers/2", responseId2, http.StatusOK},
+		{"valid imposter with id 1 and second variable 1 in path", imposters[1], "/gophers/1/1", responseId1Variable1, http.StatusOK},
+		{"valid imposter with id 1 and second variable 2 in path", imposters[1], "/gophers/1/2", responseId1Variable2, http.StatusOK},
+	}
+
+	for _, tt := range dataTest {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
```suggestion
	for _, tt := range dataTest {
	        tt := tt
		t.Run(tt.name, func(t *testing.T) {
```

For safety, until we upgrade minimum Go version from v1.21 to v1.22, where `loopvar` is fixed.